protocol-units/bridge/indexer-db/migrations/2024-10-31-112521_indexed_bridge_actions/up.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE lock_bridge_transfers (
+    id SERIAL PRIMARY KEY,
+    bridge_transfer_id BYTEA NOT NULL,
+    hash_lock BYTEA NOT NULL,
+    initiator BYTEA NOT NULL,      -- Address stored as bytes
```

Comment:
Might we want to limit this to 32 bytes?

Reply:
You can't restrict `BYTEA` like that. You can `CHAR(64)`. But, `BYTEA` is supposed to be better on disk and better for exact matches when querying on most engines.

Reply:
Ok, `BYTEA` is fine, the contract args are themselves restricted (only on eth side), we should restrict the move calls also, as an aside.